lib/Module/KInstruction.cpp

```diff
@@ -9,31 +9,110 @@
 
 #include "klee/Module/KInstruction.h"
 #include "klee/Module/KModule.h"
+#include "klee/Module/LocationInfo.h"
 
 #include "klee/Support/CompilerWarning.h"
 DISABLE_WARNING_PUSH
 DISABLE_WARNING_DEPRECATED_DECLARATIONS
+#include "klee/Support/ErrorHandling.h"
```

Comment:
Move outside the disabling warning region.

Reply:
Done